assets/charts/control-plane/calico/values.yaml

```diff
@@ -13,3 +13,6 @@ calico:
   podCIDR: 10.2.0.0/16
   networkEncapsulation: "ipipMode: Always"
   blockedMetadataCIDRs: []
+  # Lokomotive specific change.
+  # failsafeInboundHostPorts:
+  encryptPodsTraffic: false
```

Comment:
Can we name this `encryptPodTraffic` to match the name of the variable in terraform?